app/slimming/slimming.js

```diff
@@ -75,7 +75,7 @@ app.controller('GOSlimCtrl', function($scope, $location, $q,
   };
 
   var getMergedTermsAndTotal = function(terms, aspectMap) {
-    var mergedTerms = jQuery.extend(true, {}, $scope.selection);
+    var mergedTerms = jQuery.extend(true, {}, $scope.selection); //NOTE: use angular's extend, not jQuery
```

Comment:
Yes, we should avoid using jQuery, especially is the functionality is provided by angular.